docs/node_management_policy.md

```diff
@@ -0,0 +1,92 @@
+
+Use the `hzn exchange nmp new` command to generate an empty NMP. 
+
+## Definition
+Following are the fields in the JSON representation of an NMP:
+
+* `label`: A short description of the deployment policy suitable to be displayed in a UI. This field is required.
+* `description`: A longer description of the deployment policy. This field is not required.
+* `constraints`: Policy constraints as described [here](./properties_and_constraints.md) which refer to node policy properties.
+* `properties`: Policy properties as described [here](./properties_and_constraints.md) which a node policy constraint can refer to.
+* `patterns`: A list of patterns that this policy is compatible with. This field is mutually exclusive with the properties and constraints fields.
+* `enabled`: A boolean to indicate whether this NMP can be deployed to nodes or not.
+* `start`: An RFC3339 formatted timestamp for when the NMP should be executed on compatible nodes. The value "now" can be specified to indicate that the NMP shoul dbe executed immediately.
+* `startWindow`: A value used to randomize the NMP start time in order to reduce the number of nodes running the same NMP simultaneously. The NMP will start within `start` + `startWindow` seconds.
+* `agentUpgradePolicy`: A JSON structure to define an automatic agent upgrade job.
+    * `manifest`: The name of a manifest that exists in the Management Hub that describes the packages and versions that will be installed. Manifests are described in more detail [here](./agentfile_manifest.md)
+    * `allowDowngrade`: A boolean to indicate whether this upgrade job can perform a downgrade to a previous version.
+
+## Example
+
+The following is an example of a NMP json file. In this example, the properties and contraints are being used to specify deployment, so the patterns field is omitted. This NMP will be executed on the next node heartbeat since the start field is set to "now" and it has been enabled. The job being performed in this NMP is an automatic agent upgrade job, so a manifest has been specified in the agentUpgradePolicy field. This manifest should contain the software, certificate, and/or config files and versions that the agent will be upgraded to. In this case, if any of the versions specified in the manifest are lower than currently installed, they will be skipped since the allowDowngrade field is set to false.
+```
+{
+  "label": "Sample NMP",
+  "description": "A sample description of the NMP",
+  "constraints": [
+    "myproperty == myvalue"
+  ],
+  "properties": [
+    {
+      "name": "myproperty",
+      "value": "myvalue""
+    }
+  ],
+  "enabled": true,
+  "start": "now",
+  "startWindow": 0,
```

Comment:
Please give a non zero number, such as '300', so that the user will know that it's better to set it up to a number.